libafl/src/feedbacks/map.rs

```diff
@@ -154,14 +208,14 @@ where
 
 /// A filter that only saves values which are at least the next pow2 class
 #[derive(Serialize, Deserialize, Clone, Debug)]
-pub struct MaxMapOneOrFilledFilter {}
-impl<T> MapFindFilter<T> for MaxMapOneOrFilledFilter
+pub struct OneOrFilledIsNovel {}
+impl<T> IsNovel<T> for OneOrFilledIsNovel
 where
     T: PrimInt + Default + Copy + 'static + serde::Serialize + serde::de::DeserializeOwned,
 {
     #[inline]
-    fn is_interesting(old: T, new: T) -> bool {
-        (new == T::one() || new == T::one() || new == T::max_value()) && new > old
+    fn is_novel(old: T, new: T) -> bool {
+        (new == T::one() || new == T::max_value()) && new > old
```

Comment:
Technically this also makes sure the new value is larger, so it only makes sense of the values grow